sentry_sdk/integrations/django/__init__.py

```diff
@@ -325,6 +329,8 @@ def _before_get_response(request):
     integration = hub.get_integration(DjangoIntegration)
     if integration is None:
         return
+    if integration.urlconf is None:
+        integration.urlconf = import_module(settings.ROOT_URLCONF)
```

Comment:
Can we move this `import_module` call into the `except Resolver404` block down below?
So the urlconf is only imported when there is an 404 error and not for every call to _before_get_response? This would make this more scallable.

Reply:
The import should only happen once in the first request since the instance of `DjangoIntegration()` stays the same, and we check for `integration.urlconf`. I think the reason I didn't put the import in the 404 handler was to prevent any other errors if importing the URL conf fails during the 404 handling

Reply:
Just rebased it on main and your suggestion makes a lot more sense now, I've moved the import call to the 404 handler